rootfs/templates/wrapper-body.sh

```
@@ -514,36 +572,42 @@
 		--env DOCKER_NAME="${DOCKER_NAME}"
 		--env DOCKER_TAG="${DOCKER_TAG}"
-		--env GEODESIC_HOST_CWD="${GEODESIC_HOST_CWD}"
 	)
 
 	if [ "$ONE_SHELL" = "true" ]; then
 		DOCKER_NAME="${DOCKER_NAME}-$(date +%d%H%M%S)"
 		echo "# Starting single shell ${DOCKER_NAME} session from ${DOCKER_IMAGE}"
 		echo "# Exposing port ${GEODESIC_PORT}"
 		[ -z "${GEODESIC_DOCKER_EXTRA_ARGS}" ] || echo "# Launching with extra Docker args: ${GEODESIC_DOCKER_EXTRA_ARGS}"
-		docker run --name "${DOCKER_NAME}" "${DOCKER_ARGS[@]}" ${GEODESIC_DOCKER_EXTRA_ARGS} ${DOCKER_IMAGE} -l $*
+		docker run --name "${DOCKER_NAME}" "${DOCKER_LAUNCH_ARGS[@]}" "${DOCKER_EXEC_ARGS[@]}" ${GEODESIC_DOCKER_EXTRA_ARGS} ${DOCKER_IMAGE} -l $*
```

Shellcheck (0.10.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)